### DISCUSSION TITLE
Sable Admin 文档：快速开始 - WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#8] [General]
# Sable Admin 文档：快速开始 - WxW

介绍运行环境配置 nodejs 版本需要在 14.x 以上 nodejs 是完全免费的，您不必为此支付任何资金   本项目使用 pnpm 作为包管理工具，建议使用 pnpm 进行依赖的安装及其他操作  前置知识 您需要了解 Vue3 的基础知识 ts 需要了解基础部分，您可以观看本站点的 TypeScript 基础 来对 ts 进行初步了解 需要会使用 git 进行代码拉取  如何加入 sable

http://localhost:4000/wiki/sable/index.html

**Comment by ma-wenqian** [comment]
作者你好 我也用的hexo和这个主题，想请教一下你的网站是部署在阿里云的服务器还是oss内呀

**Reply by Whbbit1999** [reply]
阿里的服务器上

**Comment by Whbbit1999** [comment]
基本功能已完成
